src/components/TxData.js

```diff
@@ -35,6 +35,12 @@ class TxData extends React.Component {
    */
   state = { raw: false, children: false, tokens: [] };
 
+  // Array of token uid that was already found to show the symbol
+  tokensFound = [];
```

Comment:
I'll add a comment on the backend MR for the tx detail API. I think we can have all this information returned in the API, so there's no need to have any of this here.

Reply:
Actually I don't see how the API could help here, this information depends on the tokens registered on the wallet (information saved locally), so it would be impossible for the API to know.

Reply:
Sorry, my comment was on the wrong place. I'm not sure we have an array with all tokens, known and unknown (maybe `state.tokens`).

The idea is that we shouldn't need to call `hathorLib.walletApi.getTokenInfo`. We can return a token's name and symbol in the tx details api.

Reply:
The API was updated and the call to `hathorLib.walletApi.getTokenInfo` is not needed anymore